pkg/get/tools.go

```diff
@@ -24,9 +24,10 @@ func MakeTools() Tools {
 
 	tools = append(tools,
 		Tool{
-			Owner: "openfaas",
-			Repo:  "faas-cli",
-			Name:  "faas-cli",
+			Owner:       "openfaas",
+			Repo:        "faas-cli",
+			Name:        "faas-cli",
+			Description: "Official CLI for OpenFaaS.",
```

Comment:
Thanks for adding all these Descriptions, that must have taken you a long time. It's appreciated.

Reply:
Welcome, it was pretty fun to explore all the tools offered by arkade.